src/language-js/parse/hermes.js

```diff
@@ -31,8 +31,8 @@ function createParseError(error) {
 const parseOptions = {
   flow: "all",
   babel: false,
-  // Not supported yet
-  enableExperimentalFlowMatchSyntax: false,
+  // enableExperimentalComponentSyntax: true, // Enable by default
+  // enableExperimentalFlowMatchSyntax: true, // Enable by default
```

Comment:
What does this comment out mean?

Reply:
I want to show that these options exist, but they are enabled by default, so it's unnecessary to enable them explicitly.